### ISSUE TITLE
Line-height should be 1.5 to comply the WCAG guideline

### ISSUE THREAD

**Issue by NagayamaToshiaki** [nicolas-cusan/destyle.css#18]
destyle's base line-height is 1.15 and this violates the WCAG Success Criterion 1.4.12: Text Spacing.
To suits the criterion, line-height must be "at least 1.5 times the font size".
So, html element's style should be:
html {
  line-height: 1.5; /* 1 */
  -webkit-text-size-adjust: 100%; /* 2 */
  -webkit-tap-highlight-color: transparent; /* 3*/
}

**Comment by nicolas-cusan** [comment]
@NagayamaToshiaki sorry for the late replay. And thank you for your feedback.

The line-height should be set for each project individually. If `line-height: 1.5` is your preference you should just add that in your own stylesheet.
